compiler/src/iree/compiler/Codegen/Common/TensorToVectorVectorizePad.cpp

```diff
@@ -101,12 +101,12 @@ struct VectorizePadWithConditions final
 
     /// Return true if the given `attrOrValue` is a constant zero.
     auto isConstantZero = [](OpFoldResult attrOrValue) {
-      if (attrOrValue.is<Attribute>()) {
-        auto attr = llvm::dyn_cast<IntegerAttr>(attrOrValue.get<Attribute>());
+      if (isa<Attribute>(attrOrValue)) {
+        auto attr = dyn_cast<IntegerAttr>(cast<Attribute>(attrOrValue));
         return attr && attr.getValue().getZExtValue() == 0;
       }
       IntegerAttr attr;
-      return matchPattern(attrOrValue.get<Value>(), m_Constant(&attr)) &&
+      return matchPattern(cast<Value>(attrOrValue), m_Constant(&attr)) &&
```

Comment:
Can we avoid casting twice?

Reply:
I think once its casted to Attribute and another time to Value so I am not sure we can avoid it.